test.js

```diff
@@ -87,3 +88,12 @@ test('does not prepend newline if first word is split', function (t) {
 	t.assert(res.split('\n').length === 2);
 	t.end();
 });
+
+test('takes into account line returns inside input', function (t) {
+	var res20 = fn(fixture2, 10, {hard: true});
+	t.assert(res20 === '12345678\n9012345678\n90');
+	t.assert(stripAnsi(res20).split('\n').every(function (x) {
```

Comment:
Hmm. Not sure what you mean by that? The second (comparative) string has an added line break near the end that only this method would properly handle, I believe.